src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -169,29 +128,27 @@ function OptionRowLHN(props) {
                 {(hovered) => (
                     <PressableWithSecondaryInteraction
                         ref={popoverAnchor}
-                        onPress={(e) => {
-                            if (e) {
-                                e.preventDefault();
-                            }
+                        onPress={(event) => {
+                            event?.preventDefault();
                             // Enable Composer to focus on clicking the same chat after opening the context menu.
                             ReportActionComposeFocusManager.focus();
-                            props.onSelectRow(optionItem, popoverAnchor);
+                            onSelectRow(optionItem, popoverAnchor);
                         }}
-                        onMouseDown={(e) => {
+                        onMouseDown={(event) => {
                             // Allow composer blur on right click
-                            if (!e) {
+                            if (!event) {
                                 return;
                             }
 
                             // Prevent composer blur on left click
-                            e.preventDefault();
+                            event.preventDefault();
                         }}
                         testID={optionItem.reportID}
-                        onSecondaryInteraction={(e) => {
-                            showPopover(e);
+                        onSecondaryInteraction={(event) => {
+                            showPopover(event);
                             // Ensure that we blur the composer when opening context menu, so that only one component is focused at a time
                             if (DomUtils.getActiveElement()) {
-                                DomUtils.getActiveElement().blur();
+                                (DomUtils.getActiveElement() as HTMLElement | null)?.blur();
```

Comment:
```suggestion
                                (DomUtils.getActiveElement() as HTMLElement)?.blur();
```

Reply:
Hey @VickyStash I suggested this one 😅 it was `(DomUtils.getActiveElement() as HTMLElement)?.blur();`, but is wrong because `DomUtils.getActiveElement()` returns `Element | null`, so when asserting to `HTMLElement` we have to consider `null` too